tarn/location/interface.py

```diff
@@ -51,5 +49,19 @@ def write(self, key: Key, value: Value, labels: MaybeLabels) -> ContextManager[M
     def delete(self, key: Key) -> bool:
         pass
 
+    def touch(self, key: Key) -> bool:
+        """
+        Update usage date for a given `key`
+        """
+        return False
```

Comment:
mb also make this one abstract? The user should be at least warned if he forgets to implement this